src/components/mc/cuda/mc_cuda.c

```diff
@@ -415,11 +416,14 @@ ucc_status_t ucc_ee_cuda_task_query(void *ee_req)
 ucc_status_t ucc_ee_cuda_task_end(void *ee_req)
```

Comment:
on entry to this function the state is either COMPLETED_ACK or STARTED right?

Reply:
correct